python/pyarrow/tests/test_extension_type.py

```diff
@@ -1926,3 +1926,31 @@ def test_bool8_scalar():
     assert pa.scalar(1, type=pa.bool8()).as_py() is True
     assert pa.scalar(2, type=pa.bool8()).as_py() is True
     assert pa.scalar(None, type=pa.bool8()).as_py() is None
+
+
+@pytest.mark.parametrize("str_type", (pa.utf8, pa.large_utf8))
```

Comment:
The test doesn't work yet for `pa.utf8_view()`?

Reply:
It does, expanded the test set.